src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -494,8 +493,8 @@ ucs_status_t uct_rc_verbs_ep_flush(uct_ep_h tl_ep, unsigned flags,
     uct_rc_verbs_ep_t *ep = ucs_derived_of(tl_ep, uct_rc_verbs_ep_t);
     ucs_status_t status;
 
-    if (comp != NULL) {
-        return UCS_ERR_UNSUPPORTED;
+    if (!uct_rc_iface_has_tx_resources(&iface->super)) {
```

Comment:
Right, the ordering of these conditions does not matter.  If we have no resource, not everything is flushed yet obviously
Ok.